packages/apps/project/src/components/templates/ActivitiesListView/ActivitiesListView.tsx

```diff
@@ -0,0 +1,241 @@
+    );
+
+    if (differenceInDays > 0) {
+      return `${I18n.t('Project_UpdatedTimes', {
+        times: `${differenceInDays} ${I18n.t('Project_Day').toLowerCase()}${
+          differenceInDays > 1 ? 's' : ''
+        }`,
+      })}`;
+    } else {
+      const _differenceInMilliseconds = now.getTime() - date.getTime();
+      const differenceInHours = Math.floor(
+        _differenceInMilliseconds / (1000 * 60 * 60),
+      );
+      if (differenceInHours > 0) {
+        return `${I18n.t('Project_UpdatedTimes', {
+          times: `${differenceInHours} ${I18n.t('Project_Hour').toLowerCase()}${
+            differenceInHours > 1 ? 's' : ''
+          }`,
+        })}`;
+      } else {
+        const differenceInMinutes = Math.floor(
+          _differenceInMilliseconds / (1000 * 60),
+        );
+        return `${I18n.t('Project_UpdatedTimes', {
+          times: `${differenceInMinutes} ${I18n.t(
+            'Project_min',
+          ).toLowerCase()}${differenceInMinutes > 1 ? 's' : ''}`,
+        })}`;
+      }
+    }
+  };
+
+  const renderItem = ({item}) => {
+    const key = Object.keys(item)?.[0];
+
+    const activity = item?.[key]?.[0].activity || {};
+
+    const modelName = item?.[key]?.[0]?.modelName;
+
+    const tracks = activity?.tracks || [];
+
+    const time = item?.[key]?.[0].time || '';
+
+    const user = item?.[key]?.[0].user || '';
+
+    const utilityClass = item?.[key]?.[0].utilityClass;
+
+    return (
+      <View style={styles.item}>
+        <Text style={styles.updatedText}>{`${I18n.t(
+          'Project_UpdatedBy',
+        )} ${user} ${calculateTimeDifference(time)}`}</Text>
+        <MailMessageNotificationCard
+          relatedModel={''}
+          relatedId={0}
+          title={key}
+          tracks={tracks}
+          customTopComponent={
+            <View style={styles.headerMailMessage}>
+              <Text>{sliceString(key, 25)}</Text>
+              <Badge
+                title={modelName}
+                color={
+                  utilityClass === 'danger'
+                    ? Colors.errorColor
+                    : utilityClass === 'success'
+                    ? Colors.successColor
+                    : Colors.primaryColor
+                }
+              />
+            </View>
+          }
+        />
+      </View>
+    );
+  };
+
+  const renderSectionHeader = ({section: {title}}) => {
+    const dateToDisplay = convertToDate(title);
+
+    const isValidDate = !isNaN(dateToDisplay.getTime());
+
+    const displayDate = isValidDate ? dateToDisplay : new Date();
+
+    return (
+      <View style={styles.header}>
+        <DateDisplay date={displayDate.toString()} displayYear={true} />
+      </View>
+    );
+  };
+
+  return (
+    <SectionList
+      sections={dataList}
+      keyExtractor={(item, index) => item + index}
+      renderItem={renderItem}
+      renderSectionHeader={renderSectionHeader}
+      onEndReached={handlePreviousActivity}
+    />
```

Comment:
We also need to have the pull to refresh function to reset date to today and refresh first data